### ISSUE TITLE
Maintainer information?

### ISSUE THREAD

**Issue by RemiTorracinta** [Legrandin/pycryptodome#438]
Hello,

I'd like to use pycryptodome at my current organization. I'm (reasonably) expected to do due diligence on the maintainers of the project. I found it difficult to find any public information about maintainers of this project which would engender some trust. 

Ideally it would be possible to find some evidence of the competence and good intentions of the maintainer(s), as I'm naturally not in a position to evaluate the cryptographic primitives myself.

Is the information I'm looking for available somewhere?

**Comment by frispete** [comment]
While there isn't much available about Helder Eijs than what's here on GH, Helder appears to be pretty responsive when dealing with issues since starting with this project. 

Please take into account, that maintaining such a project isn't always fun, while there isn't any payment to compensate. For the most part, this is due to insulting attacks. 

Similar to security technology, the more you reveal about yourself, the greater the public attack vector. 

Just do what I do and watch anything in this space for a longer time (I started about 4 years ago). There is no evidence to cast doubt on his reliability. On the contrary, Helder and his companions do a great job and leave nothing to be desired (and if they do, the reasons can be understood).

As a package maintainer for a lot of python packages in the openSUSE eco system, I can assure you that the code quality of this project is significantly higher than other security related packages, I've maintained so far..

**Comment by RemiTorracinta** [comment]
Thank you, frispete, for these comments. 

To be sure, Helder is free to stay pseudonymous or minmize publicly available information. I think this kind of thing can affect the acceptability of the package in in some enterprise environments, but adoption there doesn't have to be a goal of the project.

A good alternative in my mind is the network of trust. If trusted users or organizations (maybe like yourself) publicly confirm that they use the project (validating it however they deem necessary, or by accessing non-public information), that makes it easier to trust the project.

I'll close this ticket for now as I don't think a resolution to the original request is likely.

**Comment by frispete** [comment]
> To be sure, Helder is free to stay pseudonymous or minmize publicly available information. I think this kind of thing can affect the acceptability of the package in in some enterprise environments, but adoption there doesn't have to be a goal of the project.

@RemiTorracinta I think, that the subordinate clause is mispronounced.

> A good alternative in my mind is the network of trust. If trusted users or organizations (maybe like yourself) publicly confirm that they use the project (validating it however they deem necessary, or by accessing non-public information), that makes it easier to trust the project.

I hereby confirm, that I use this project in both, publicly available Open-Source as well as safety-related inhouse projects of various organizations. While I'm not a cryptographer, I based my decision on code examination, conformity, responsiveness, sustainability, dependencies, stability over time and licenses. During that course, I followed many commits closely, and tried to value Helder's decisions. I can't recall an instance where I found his decisions questionable in any respect.

A few more words on security in Open-Source: publicly available source code doesn't mean better security out of the box. There is much more to it than that. 

See the PyYAML disaster for example - which is actively exploited for attacks on enterprise infrastructure of all kind right now. The developers are talking themselves out of the fact that they have long warned about the unsafe API calls in the documentation, instead of marking their high-risk interfaces accordingly to force their API users to make decisions in this regard.

PyCrypto, the predecessor of PyCryptodome had also a significant record of security related and other issues, that Helder inherited. Before his first release, he fixed many of them. Since then, it's a pleasure to see arising issues handled quickly and properly, always weighting security against backward compatibility carefully.

Both, PyYAML and PyCrypto reveal a lot of private information about their creators. If that's the reason both are still being used in enterprise environments in *unsafe* conditions, the trust decision process is pretty damaged.
